### ISSUE TITLE
Statev UI verursacht FPS Drops

### ISSUE THREAD

**Issue by powerswissi** [mexyn/statev_v2_issues#1905]
## WICHTIGE INFORMATION!!:
```
Hier sollen nur Bugs gemeldet werden, Vorschläge bitte ins StateV-Forum!
```

**Character Name**

Roger Thickle
Sophia Black 
Jason_Bean

**Auftrittszeitpunkt (Datum / Uhrzeit)**

seit dem altv update

**Beobachtetes Verhalten**

FPS Drops beim Fahren bei eingeschalteter UI von State. 
Auf dem Ölfeld sind normaler weise 80 FPS plus beim Fahren. Seit dem altv update sind wir bei 50 -60

**Erwartetes Verhalten**

Bei eingeschaltetem UI sollten sich die FPS nicht verändern. Bzw nicht gross verändern!

**Schritte um den Fehler nachvollziehen zu können**

Einloggen 
mit einem Auto / LKW fahren und UI an und abschalten mit #

**Monitorauflösung (nur wenn Falsche Darstellung in der UI)**

**Optional: Video / Bilder des Fehlers**

https://clips.twitch.tv/HedonisticBoringZebraFeelsBadMan-wjEeGBakWJZBMLjP

**Comment by JannikDatsch** [comment]
Jo, deswegen fahre ich zum Teil in der Stadt einfach ohne Overlay was zwar auch blöd ist aber naja. Vor allem ist dadurch auch die Map und die Essensanzeige weg.  Von daher von mir der Vorschlag, dass es eine weitere "#" Stufe gibt, in der man nur das Tacho UI wegschalten kann, und Essen, Funk und Map bleiben, denn soweit ich weiß, ist der Tacho das absolute Hauptproblem

--> 1. Gesamtes UI da
--> 2. Gesamtes UI außer Tacho da
--> 3. Gesamtes UI weg
